server/build/webpack.js

```diff
@@ -174,6 +174,9 @@ export default async function createCompiler (dir, { buildId, dev = false, quiet
     // That's why we need to do this.
     const { options } = externalBabelConfig
     mainBabelOptions.babelrc = options.babelrc !== false
+    if (!mainBabelOptions.babelrc) {
+      Object.assign(mainBabelOptions, externalBabelConfig.options)
```

Comment:
Lets make this `mainBabelOptions = {... mainBabelOptions, ...externalBabelConfig.options}` and make `mainBabelOptions` a `let`